docs/userguide.md

```diff
@@ -777,9 +777,8 @@ spec:
 
 ### `spec.placement.clusters` is not provided, `spec.placement.clusterSelector` is provided but empty
 
-In this case, `spec.placement.clusterSelector` will be ignored, since
-`spec.placement.clusters` is provided. This ensures that the results of runtime
-scheduling have priority over manual definition of a cluster selector.
+In this case, `spec.placement.clusters` will be ignored, since only
+`spec.placement.clusterSelector` is provided.
```

Comment:
This is covered by https://github.com/kubernetes-sigs/kubefed/pull/1239/files#diff-63246bcc406b02703a4bbc30c80e355dR789 so we can remove this completely.

```suggestion
```